client/src/assets/scss/foundation/_settings.scss

```diff
@@ -297,11 +297,11 @@ $button-font-weight: null;
 $button-padding: 0.85em 1em;
 $button-margin: 0 0 $global-margin 0;
 $button-fill: solid;
-$button-background: $primary-color;
+$button-background: #f17b30;
 $button-background-hover: scale-color($button-background, $lightness: -15%);
 $button-color: $white;
 $button-color-alt: $black;
-$button-radius: $global-radius;
+$button-radius: 20px;
```

Comment:
Let's not update these Foundation settings, this should be our source of truth for Foundation styles. Instead, you can overwrite any styles within your `main.scss` or create any custom styles

Reply:
I see, thank you for the clarification!